src/components/Filters/CheckboxOption.tsx

```diff
@@ -97,14 +97,14 @@ export function CheckboxOption(props: CheckboxOptionProps): JSX.Element | null {
     handleClick(evt.target.checked);
   }, [handleClick]);
 
-  const optionFilter: Filter = useMemo(() => {
+  const optionFilter: FieldValueFilter = useMemo(() => {
```

Comment:
Is this a FieldValueFilter if it's missing the `kind: 'fieldValue'` property? I wonder if it's simpler to put the fieldValue here

Reply:
Should we change the kind value to something other than `"fieldValue"` then? Seems a little confusing to me that a Filter with kind = fieldValue isn't a FieldValueFilter

Reply:
I'm not sure I follow. the "Filter" model no longer exists because it was renamed to `FieldValueFilter` (which has the same `fieldId`, `matcher`, and `value`). now, if you want to specify a static filter with a `kind` of fieldValue, that would be a `FieldValueStaticFilter`

Reply:
Can we change the "fieldValue" kind to "fieldValueStaticFilter" or something that makes it clear the associated model isn't `FieldValueFilter` but `FieldValueStaticFilter`? I didn't notice this until seeing the models used in the component lib. Seeing `kind: 'fieldValue'` makes me think "oh this object is a FieldValueFilter" when in reality it's a FieldValueStaticFilter.

Reply:
Ideally I think it would be good if we came up for different names so that it's easier to differentiate FieldValueFilter and FieldValueStaticFilter but I don't have any ideas and this could be a "me" thing

Reply:
right now, the `kind` values are consistent so that `FieldValueStaticFilter` has `kind: 'fieldValue'`, `ConjunctionStaticFilter` has `kind: 'conjunction'`, and `DisjunctionStaticFilter` has `kind: 'disjunction'`. would it be more clear if each `kind` was the full type (e.g. `kind = 'fieldValueStaticFilter' | 'conjunctionStaticFilter' | 'disjunctionStaticFilter'`)?

Reply:
I was thinking about it more as what kind of static filter it is, rather than what the corresponding type is. So the other data that you'll be able to access for the filter would correspond to a 'fieldValue' or to a 'conjunction'

Reply:
Ah I see. I think I can get used to the naming! I was trying to think to alternatives to `FieldValueFilter` but I could only come up with stuff like `ApiFilter` and `BaseFilter`